elixir/apps/api/lib/api/gateway/channel.ex

```diff
@@ -648,52 +648,6 @@ defmodule API.Gateway.Channel do
     end
   end
 
-  def handle_in(
```

Comment:
@thomaseizinger Just confirming - the Gateway doesn't send any form of metrics today correct?